lib/vdsm/virt/vmdevices/lease.py

```diff
@@ -138,6 +138,7 @@ class Device(core.Base):
     """
     VM lease device.
     """
+    # pylint: disable=no-member
```

Comment:
There is no failure on master:
https://github.com/oVirt/vdsm/actions/runs/3113779231/jobs/5048829937

And failure in `__slots__` definition is very unlikely, this line does not
try to access any member.

Please explain how to reproduce all the errors fixed by this commit, and share
the error message from pylint for every line fixed.

It is also not clear why this change is related to the spdx change. If we have
an issue, it should be addressed by another pr.

Reply:
The problem appeared in this branch, not sure exactly on which commit but I assume it is related with updating the `pylintrc` to add the license.

I will split the commit into another PR, although that would create a (yet another) dependency.

The failures are not in the lines where the rules are disabled, the rules are disabled for the whole class. In the commit message I have included one example, all looked the same for all three classes involved.